stellarphot/photometry.py

```diff
@@ -636,49 +735,4 @@ def find_times(phot_column, exposure,
 
     new_time = bary_time.jd
 
-    return new_time
-
-
-# ra=331.1170417, dec=81.5659444, latitude=46.86678, longitude=263.54672
-def find_bjd_mid_exposure(utc_times, exposure=0,
-                          coords=None, location=None):
-    """
-    Returns a numpy array of barycentric Julian date times
-
-    Parameters
-    ----------
-
-    phot_column : `astropy.table.Column` of `astropy.times.Time`  or numpy array
-        numpy array or column of observation dates/times.
-
-    exposure : float; optional
-        exposure time in seconds
-
-    coords : `astropy.coordinates.SkyCoord`
-        Ra/Dec of the object to be used for the light travel time
-        calculation.
-
-    location: `astropy.coordinates.EarthLocation`
-        Location of the observatory.
-
-    Returns
-    -------
-
-    new_time : numpy array
-        array of barycentric times by Julian date
-
-    """
-    location = EarthLocation(lat=latitude, lon=longitude)
-    ip_peg = SkyCoord(ra=[ra], dec=[dec], unit='degree')
-
-    if coords is None or observatory_location is None:
-        raise ValueError
-    times = Time(phot_column, scale='utc', format='isot', location=location)
-    ltt_bary = times.light_travel_time(coords)
-    times_tdb = times.tdb
-    time_barycenter = times_tdb + ltt_bary
-
-    # adjust to midpoint of exposure
-    bary_time = time_barycenter + exposure * u.second / 2
-
-    return bary_time.jd
+    return new_time
```

Comment:
```suggestion
    return new_time

```

Reply:
I tweaked the code to remove the un-necessary `new_time` and returned `bary_time.jd` directly.  Also tweaked docstring to match this.